BigBaseV2/src/hooking.cpp

```diff
@@ -59,7 +59,12 @@ namespace big
 		// Player Has Joined
 		m_player_has_joined_hook("PHJ", g_pointers->m_player_has_joined, &hooks::player_join),
 		// Player Has Left
-		m_player_has_left_hook("PHL", g_pointers->m_player_has_left, &hooks::player_leave)
+		m_player_has_left_hook("PHL", g_pointers->m_player_has_left, &hooks::player_leave),
+
+		// Chat recieve
+		m_chat_receive_hook("CR", g_pointers->m_chat_receive, &hooks::chat_receive),
```

Comment:
recieve is incorrect, it should be receive

Reply:
It already is